sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/FeatureManagementConfiguration.java

```diff
@@ -19,7 +19,7 @@ class FeatureManagementConfiguration {
 
```

Comment:
To create a feature flag that is enabled and a list of feature flags that is empty currently requires:

```yml
feature-management:
  FeatureFlags:
    -
      id: Alpha
      enabled: true
      conditions:
        client_filters: {}
```

Which is a bit weird.
Note: This is expected to return false, which it does not do.

Reply:
Yes. I agree that it's a nit wired. I used to think about to support both `array` and `Map<Integer, FeatureFilterEvaluationContext>` for the "client_filters" property. Would it be better?
If we support array, then we could set an empty list with the following
```
feature-management:
  FeatureFlags:
    -
      id: Alpha
      enabled: true
      conditions:
        client_filters: []
```
And also we should support the map.
```
FeatureManagement:
    FeatureFlags:
        -
            id: Alpha
            enabled: true
            conditions:
                client_filters:
                    -
                        name: Microsoft.Random
                        parameters:
                            Value: 50
                    -   
                        name: Microsoft.TimeWindowFilter
                        parameters:
                            Start: "Thu, 29 Jun 2023 07:00:00 GMT"
                            End: "Wed, 30 Aug 2023 07:00:00 GMT"
```

Reply:
I may be missing something, but both of those are lists for the client_filters field.

Reply:
If it's no a new dependency to the SKD library that I'm good with it.